docker/dev.Dockerfile

```diff
@@ -33,10 +33,22 @@ ENV SHELLCHECK_VERSION="0.4.6"
 RUN curl -sSfL "https://storage.googleapis.com/shellcheck/shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" \
     | tar -Jx --strip-components=1 -C /usr/local/bin
 
+#Poetry Installation
+ENV POETRY_VERSION=1.0.5
+RUN pip install "poetry==$POETRY_VERSION"
+
 # /app will be mounted via a volume defined in docker-compose
 ADD . /app
 WORKDIR /app
 
+COPY poetry.lock pyproject.toml /app/
+RUN pip install --no-cache-dir --upgrade pip \
+    && pip install --no-cache-dir poetry \
```

Comment:
Why installing poetry a second time?

Reply:
the last command is needed to install dev and docs using poetry